### ISSUE TITLE
Using cash in vue-script2

### ISSUE THREAD

**Issue by vad11** [taoeffect/vue-script2#11]
I write in html code 
`<img src="main.js" style=display:none;>`
and
 ```
mounted() {
   Script2.load('main.js').then(function () {
      console.log( "Loaded !" );
     })
}
```
I test this in chrome dev tools and not see that script loaded from cash.

**Comment by taoeffect** [comment]
If you mean *cache*.. I have no idea what you're trying to do here, or why you've assigned a script to an `<img>` tag. :-\

**Comment by vad11** [comment]
I want load script without running, and load script after this from vue after dom maked and run it. But I want load in vue from cash from previous loading by html.

**Comment by taoeffect** [comment]
Hmm, I have no idea how to do that, sorry.

Why not just put the script's functionality into a function, and then call the function when you need it?

**Comment by vad11** [comment]
Yes, I do this. But loading script take many time (100 -200 ms). And I want load script before in do nothing time.
